app/views/layouts/application.html.erb

```diff
@@ -1,4 +1,3 @@
-<% content_for :app_title, 'GOV.UK Signon' %>
 <% content_for :page_title, "#{yield(:title)} | GOV.UK Signon" %>
```

Comment:
I wasn't going to alter it as part of this PR. I think it's something that's needed in the admin template itself.